Tests/Editor/GameLiftPlugin/AnywherePageTests.cs

```diff
@@ -0,0 +1,176 @@
+                .Verifiable();
+
+            return new AwsCredentials(creationMock.Object, updateMock.Object, coreApi);
+        }
+
+        private GameLiftRequestAdapter ArrangeAnywhereFleetHappyPath()
+        {
+            var listLocationModel = new List<LocationModel>();
+            listLocationModel.Add(new LocationModel()
+            {
+                LocationName = "custom-location-1"
+            });
+
+            _coreApi.Setup(f => f.PutSetting(It.IsAny<string>(), It.IsAny<string>())).Returns( AmazonGameLiftPlugin.Core.Shared.Response.Ok<PutSettingResponse>(new PutSettingResponse()));
+            _coreApi.Setup(f => f.PutSetting(It.IsAny<string>(), null)).Returns( AmazonGameLiftPlugin.Core.Shared.Response.Fail<PutSettingResponse>(new PutSettingResponse()));
+            _coreApi.Setup(f => f.PutSetting(It.IsAny<string>(), string.Empty)).Returns( AmazonGameLiftPlugin.Core.Shared.Response.Fail<PutSettingResponse>(new PutSettingResponse()));
+            _gameLiftWrapperMock.Setup(wrapper => wrapper.ListLocations(It.IsAny<ListLocationsRequest>())).Returns(Task.FromResult(new ListLocationsResponse(){Locations = listLocationModel}));
+            
+            _gameLiftWrapperMock.Setup(wrapper => wrapper.CreateFleet(It.IsAny<CreateFleetRequest>())).Returns(Task.FromResult(
+                new CreateFleetResponse()
+                {
+                    FleetAttributes = new FleetAttributes() { FleetId = "test" }, 
+                    LocationStates = new List<LocationState>()
+                }));
+
+            _amazonGameLiftClientFactory.Setup(f => f.Get(It.IsAny<string>()))
+                .Returns(_gameLiftWrapperMock.Object);
+
+            _awsCredentialsFactory.Setup(f => f.Create())
+                .Returns(GetAwsCredentialsWithStubComponents(_coreApi.Object));
+            
+            var gameLiftPlugin = new GameLiftPlugin(_awsCredentialsFactory.Object, _coreApi.Object, _amazonGameLiftClientFactory.Object);
+            gameLiftPlugin.CurrentState = new GameLiftPlugin.State() { SelectedProfile = "unitTest" };
+            gameLiftPlugin.SetupWrapper();
+
+            return new GameLiftRequestAdapter(gameLiftPlugin);
+        }
+        
+        [Test]
+        public void CreateAnywhereFleet_CorrectInputs_ExpectSuccess()
+        {
+            //Arrange
+            var gameLiftRequestAdapter = ArrangeAnywhereFleetHappyPath();
+            
+            //Act
+            var createFleetResult =  gameLiftRequestAdapter.CreateAnywhereFleet("test").GetAwaiter().GetResult();
+            
+            //Assert
+            _gameLiftWrapperMock.Verify(wrapper => wrapper.CreateFleet(It.IsAny<CreateFleetRequest>()), Times.Once);
+            _coreApi.Verify(f => f.PutSetting(It.IsAny<string>(), It.IsAny<string>()), Times.Exactly(2));
+            
+            Assert.IsTrue(createFleetResult);
+        }
+        
+        [Test]
+        public void CreateAnywhereFleet_NullWrapper_ExpectFalse()
```

Comment:
Please clarify your results. At a minimum `ExpectFailed` instead of `ExpectFalse` as a boolean is not obvious from a method name like `CreateAnywhereFleet`; however, it's preferred that it is more explicit.

Example for this test `CreateAnywhereFleet_WhenNullWrapper_DoesNotCallCreate`